PYME/experimental/_triangle_mesh.pyx

```diff
@@ -797,11 +837,6 @@ cdef class TriangleMesh(TrianglesBase):
         #self._E = None
         #self._pE = None
 
-    def _clear_flags(self):
```

Comment:
This is / was used in ch_shrinkwrap (hence the deprecation rather than deletion

Reply:
OK - just seen that it's changed in ch-shrwinkwrap#22 - do we want to leave the deprecated version here for a bit in case people are on differing versions?

Reply:
I think it calls a function that no longer exists?

Reply:
Fine to leave it, though.

Reply:
ahh, it was supposed to call `_invalidate_cached_properties()` as a fallback for the renaming, but I obviously got the naming wrong (not sure how my ch-shrinkwrap still works if that's the case ...)

Reply:
At this point it's probably easier to just delete than fix